mne/viz/_3d.py

```diff
@@ -584,7 +584,7 @@ def _sensor_shape(coil):
         circle *= coil['size'] / 2.  # radius of coil
         rrs = np.array([circle.real, circle.imag]).T
         tris = _make_tris_fan(n_pts + 1)
-    elif id_ in (4002, 5001, 5003, 5004, 4004, 4005, 6001, 7001):
+    elif id_ in (4002, 5001, 5003, 5004, 4004, 4005, 6001, 7001, 10001, 10002):
```

Comment:
This would be more DRY and clearer to use the FIFF constant name, but since we were lazy about it before you don't have to change it